requirements.txt

```diff
@@ -2,6 +2,7 @@ bigcode-eval@git+https://github.com/bigcode-project/bigcode-evaluation-harness.g
 click
 deepdiff
 deepeval==1.4.0
+docker
```

Comment:
why add docker as a dependency?

Reply:
hi, metrics_docker.py need the docker package :  https://github.com/opea-project/GenAIEval/blob/main/evals/benchmark/stresscli/commands/metrics_docker.py#L11